node/coinstacks/common/api/src/evm/service.ts

```diff
@@ -357,16 +357,45 @@ export class Service implements Omit<BaseAPI, 'getInfo'>, API {
       gasUsed: tx.ethereumSpecific.gasUsed?.toString() ?? '0',
       gasPrice: tx.ethereumSpecific.gasPrice.toString(),
       inputData: inputData && inputData !== '0x' && inputData !== '0x0' ? inputData : undefined,
-      tokenTransfers: tx.tokenTransfers?.map((tt) => ({
-        contract: tt.contract,
-        decimals: tt.decimals,
-        name: tt.name,
-        symbol: tt.symbol,
-        type: tt.type,
-        from: tt.from,
-        to: tt.to,
-        value: tt.value,
-      })),
+      tokenTransfers: tx.tokenTransfers?.map((tt) => {
+        const value = (() => {
+          switch (tt.type) {
+            case 'ERC721':
+            case 'BEP721':
+              return '1'
+            case 'ERC1155':
+            case 'BEP1155':
+              return tt.multiTokenValues?.[0].value ?? '0'
+            default:
+              return tt.value
+          }
+        })()
+
+        const id = (() => {
+          switch (tt.type) {
+            case 'ERC721':
+            case 'BEP721':
+              return tt.value
+            case 'ERC1155':
+            case 'BEP1155':
+              return tt.multiTokenValues?.[0].id
```

Comment:
get multi token id (not sure why it was set up as an array on the blockbook side, only ever length of 1 at this time)

Reply:
Ditto, referring to trezor-suite as the source of truth 0 looks correct, though we might want to guard against `multiTokenValues[0]` being defined, or alternatively, `multiTokenValues?.length`

Reply:
doin it